consensus/types/src/chain_spec.rs

```diff
@@ -461,6 +462,18 @@ impl ChainSpec {
         Hash256::from(domain)
     }
 
+    pub fn maximum_gossip_clock_disparity(self) -> Duration {
+        Duration::from_millis(self.maximum_gossip_clock_disparity_millis)
+    }
+
+    pub fn ttfb_timeout(self) -> Duration {
+        Duration::from_secs(self.ttfb_timeout)
+    }
+
+    pub fn resp_timeout(self) -> Duration {
+        Duration::from_secs(self.resp_timeout)
+    }
+
```

Comment:
all of these are consuming the `ChainSpec`, which is unnecessary. If you change these to be `&self` instead of `self`  you can remove all the `clone` you just added